DIMS.nf

```diff
@@ -28,7 +28,8 @@ include { GenerateBreaks } from './CustomModules/DIMS/GenerateBreaks.nf' params(
 include { GenerateExcel } from './CustomModules/DIMS/GenerateExcel.nf' params(
     analysis_id:"$params.analysis_id", 
     zscore:"$params.zscore", 
-    matrix:"$params.matrix"
+    matrix:"$params.matrix",
+    sst_components_file:"$params.sst_components_file"
```

Comment:
why would you define parameters during process import?  I would definitely keep them separate for clarity, especially since you also define process-specific parameters in `DIMS.config`